### ISSUE TITLE
UserWarning: Could not parse beam information from header. For near-IR spectral cubes.

### ISSUE THREAD

**Issue by lib-j** [radio-astro-tools/spectral-cube#560]
Hi, 

I am using spectral-cube to analyse a near-IR spectral cube from KMOS and keep getting an error message about not parsing beam information from header. This seems to assume  all spectral cubes are radio based.  Could a check be added to see if the data is in sub-mm/radio data before giving this warning?  Or a way for the user to flag no beam information is present.

anaconda3/envs/astro36/lib/python3.6/site-packages/spectral_cube/spectral_cube.py:246: UserWarning: Could not parse beam information from header.  Exception was: NoBeamException('No BMAJ found and does not appear to be a CASA/AIPS header.',)
  "  Exception was: {0}".format(ex.__repr__()))

**Comment by keflavich** [comment]
@lib-j Thanks.  Yes, we can just remove this warning, it is not terribly helpful for anyone as is, I think.

In the meantime, you can suppress the warning by doing something like:
```
import warnings
warnings.filterwarnings('ignore', module=spectral_cube)
```
I think... I'll have to check the exact syntax

**Comment by keflavich** [comment]
Looks like the appropriate syntax may (also?) be:
```
warnings.filterwarnings(action='ignore', category=spectral_cube.utils.BeamWarning)
```

**Comment by lib-j** [comment]
Fantastic. Thank you :)

**Comment by keflavich** [comment]
Let's leave this open until we actually remove the warning, though

**Comment by e-koch** [comment]
We could remove this warning in favour of one for when `SpectralCube.beam` is `None`.

**Comment by keflavich** [comment]
Yeah, I like that idea.  If you want to do it, please go ahead, otherwise I'll try to remember to try tomorrow....

(still trying to get CI to pass though =( )

**Comment by keflavich** [comment]
closed by #561
